python/argo/interaction.py

```diff
@@ -81,21 +94,36 @@ def result(self) -> Any:
         pass
 
     @abstractmethod
-    def process_result(self, result : Any) -> Any:
+    def stdout(self) -> str:
```

Comment:
I think the convention for abstract methods is to have `...` as the body, rather than `pass`. I think I put the `pass`es in earlier, but we might as well fix it now.